config/locales/devise.de.yml

```diff
@@ -93,3 +93,4 @@ de:
       not_saved:
         one: '1 Fehler verhinderte das Speichern von %{resource}'
         other: '%{count} Fehler verhinderten das Speichern von %{resource}'
+      pwned_password: "ist zuvor in einem Datenverstoß aufgetreten und sollte niemals verwendet werden. Wenn Sie es schon einmal benutzt haben, ändern Sie es sofort!"
```

Comment:
I pretty used Google Translate for all of these messages so if any native speakers want to give feedback on any or fix some in the future, that would help 👍 (I can speak and read Vietnamese, but my grammar and spelling are bad).